HOW_WE_CHANGELOG.md

```diff
@@ -154,3 +155,30 @@ These are the ones that change behavior as mentioned above. Please read [The Bre
 - dropping ruby support for x.y
 - `check-something-shiny.rb`: changed the defaults for `--lighting` from 3 to 5 as this is a better sane default. (@mygithubuser)
 ```
+
+#### Security Updates
+
+These are not new security features but intended to be used to quickly prioritize installing new security update to patch vulnerable systems. Please see https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md#security to understand our stance on versioning these changes so you understand how maintainers will handle it post merge/acceptance.
```

Comment:
Please link to the URL instead of having it inline -- something like `[our documentation on versioning security issues](https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md#security)`